server/handlers/handlers.go

```diff
@@ -616,6 +625,27 @@ func (ah *ActivityHandler) handleReactions(postID, channelID string, isDirectOrG
 }
 
 func (ah *ActivityHandler) handleDeletedActivity(activityIds clientmodels.ActivityIds) string {
+	if ah.plugin.GetSelectiveSync() {
```

Comment:
It looks like this was a gap in selective sync before, in that with the feature off we would still have deleted messages that had previously been synced. I'm wondering if, for simplicity, we "don't solve this" and leave as-is?

Reply:
Ok..I'll leave as is.